db/fixtures/miq_product_features.yml

```diff
@@ -5475,20 +5475,12 @@
         :description: Restart the Guest OS on VMs
         :feature_type: control
         :identifier: vm_guest_restart
-      - :name: Console using MKS
+      - :name: Console using HTML5
         :description: Open a web-based console for VMs
         :feature_type: control
-        :identifier: vm_console
-      - :name: Console using WebMKS
-        :description: Open a web-based console for VMs
-        :feature_type: control
-        :identifier: vm_webmks_console
```

Comment:
Do we need a data migration to move these old product features into the new names?

Reply:
I am not sure about htis part, the mapping of these features is not really 1:1 and I am afraid that we might enable something unwanted for someone. So my idea is to have a conservative approach, I asked some people in an e-mail about this, gonna it to you.

Reply:
@Fryguy do we need to have a migration if I we'd like to change these features in the default roles only? As we discussed in the e-mail with PM, we want a little conservative approach here but I'm not sure how the default roles get updated.

Reply:
As far as I tested, we don't need a migration. The seeding takes care of it from the `miq_user_roles.yml`.

Reply:
yes, It looks that we don't need migration - it is handled by `seed_feature` [method](https://github.com/ManageIQ/manageiq/blob/ad6d6751dec20b740ec4008bec4fe35f63525974/app/models/miq_product_feature.rb#L217)